packages/core/tdoptions.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Would it be possible to have only one tdoptions.json in the root of packages/, kinda like tslint.json? i.e the yarn script would call the doc generator with the options behind ../tdoptions.json from all packages.

We could also have the generated docs folder per packages (i.e package/core/docs/... and package/editor/docs/...)